packages/kit/src/views/Staking/components/ApproveBaseStake/index.tsx

```
@@ -263,13 +248,11 @@ export function ApproveBaseStake({
       permitParams.approveType = 'permit';
 
       if (permitSignatureRef.current) {
-        const amountBN = BigNumber(amount);
-        if (permitSignatureAmountRef.current) {
-          const allowanceBN = BigNumber(permitSignatureAmountRef.current);
+          const amountBN = BigNumber(amount);
```

ESLint

[error] Delete `··`

(prettier/prettier)